src/20250324_AaveV3Ethereum_OnboardEBTCAndAddEBTCWBTCEMode/OnboardEBTCAndAddEBTCWBTCEMode.md

```diff
@@ -0,0 +1,73 @@
+
+Both [TEMP CHECK](https://governance.aave.com/t/temp-check-onboard-enable-ebtc-wbtc-liquid-e-mode-on-aave-v3-core-instance/19969) and [TEMP CHECK Snapshot](https://snapshot.box/#/s:aave.eth/proposal/0x60c360b61812b9ae96b2f785f9fca7a5461ab45e295f55a695638aef18d96d50) have passed recently.
+
+## Motivation
+
+The motivation behind this proposal stems from several key factors:
+
+- High Utilization: eBTC/WBTC has demonstrated significant usage as collateral for borrowing stablecoins on the platform.
+- Capital Efficiency: Enabling liquid E-Mode for eBTC/WBTC will allow borrowers to substantially improve their capital efficiency when using this asset as collateral.
+- Controlled Growth: Liquid E-Mode provides a mechanism for more precise control over the growth and borrow demand in relation to the overall stablecoin liquidity within Aave v3 on Core Instance.
+- Enhanced Borrowing Capacity: This change will enable users to borrow larger amounts of other stablecoins against their eBTC/WBTC collateral, potentially increasing platform utilization and revenue.
+
+By implementing this proposal, we aim to optimize the use of eBTC/WBTC within the Aave ecosystem, attracting more liquidity.
+
+## Specification
+
+The table below illustrates the configured risk parameters for **eBTC**
+
+| Parameter                | Value                                      |
+| ------------------------ | ------------------------------------------ |
+| Network                  | Ethereum                                   |
+| Isolation Mode           | No                                         |
+| Borrowable               | No                                         |
+| Collateral Enabled       | Yes                                        |
+| Supply Cap               | 750                                        |
+| Borrow Cap               | -                                          |
+| Debt Ceiling             | -                                          |
+| LTV                      | 67%                                        |
+| LT                       | 72%                                        |
+| Liquidation Bonus        | 10%                                        |
+| Liquidation Protocol Fee | 10%                                        |
+| Variable Base            | -                                          |
+| Variable Slope1          | -                                          |
+| Variable Slope2          | -                                          |
+| Uoptimal                 | -                                          |
+| Reserve Factor           | -                                          |
+| Flashloanable            | No                                         |
+| Siloed Borrowing         | No                                         |
+| Oracle                   | 0x95a85D0d2f3115702d813549a80040387738A430 |
```

Comment:
hey, overall looks fine but the aip https://github.com/bgd-labs/aave-proposals-v3/pull/653/files#diff-0b89720e50fe79f3fbf2e56517c7476cb7ff2cd7f14914dbc8672a0cbff6d8f5R29-R49 does not match the configuration.
e.g. you have a borrowCap & rf, slopes.

Personally i think it's reasonable to add some values here as you did - even if not being borrowable - , but should be added on the aip.

Reply:
Done https://github.com/bgd-labs/aave-proposals-v3/pull/653/commits/fd31d459b576b83928eddff9d8766ce41ca00045